invenio_communities/communities/resources/config.py

```diff
@@ -87,7 +87,7 @@ class CommunityResourceConfig(RecordResourceConfig, ConfiguratorMixin):
 
     routes = {
         "list": "/communities",
-        "item": "/communities/<pid_value>",
+        "item": "/communities/<pid_value>/records",
```

Comment:
question: we are a bit out of the loop here, but doesn't this change one of the main endpoints of the resource? Did you consider creating a new endpoint?